Package.swift

```diff
@@ -12,13 +12,17 @@
 */
 
 import PackageDescription
+import class Foundation.ProcessInfo
 
 let package = Package(
     name: "SymbolKit",
     products: [
         .library(
             name: "SymbolKit",
             targets: ["SymbolKit"]),
+        .executable(
```

Comment:
Maybe we should start with keeping `dump-unified-graph` as an internal target and not vending it as a product? Since it's mainly useful as a debugging tool within the repo for now.